cephci.yaml.template

```diff
@@ -19,6 +19,13 @@
 #  username: <user_name>
 #  password: <user_password>
 
+# Provide stage_credentials before running suites which needs
+# access to all rhel and ceph repos
```

Comment:
```suggestion
# access to rhel and ceph repos that are configured on the staging CDN.
```

Reply:
As a new user, how would I determine whether I need to configure this option or not? In other words, what should I do in order to determine if my suite needs this?

Reply:
`All suites use staging CDN unless is_live config (or is_production) is set to true` 
will add it in template file